src/etc/inc/dyndns.class

```diff
@@ -995,6 +999,76 @@
 					curl_setopt($ch, CURLOPT_URL, $url);
 					curl_setopt($ch, CURLOPT_POSTFIELDS, $post_data);
 					break;
+				case 'azurev6':
+				case 'azure':
+					$hostname = "{$this->_dnsHost}";
+					$resourceid = trim($this->_dnsZoneID);
+					$app_id = $this->_dnsUser;
+					$client_secret = $this->_dnsPass;
+					$newip = $this->_dnsIP;
+					$newttl = $this->_dnsTTL;
+					// correct missing / in resourceid
+					if ($resourceid[0] != '/')
```

Comment:
You are checking the first character of the string, and if it isn't a `/` then you add `/` to the end of the string. Shouldn't that be consistent? (Test the end and add at the end, or test the beginning and add at the beginning).

There are probably easier ways to accomplish this depending on what you're after. For example, if it always must end with exactly one slash, you can do something like this: `rtrim($resourceid, '/') . '/'`. If it must always start and end with a slash, then `'/' . trim($resourceid, '/') . '/'`.

Reply:
thx you are right fixed along with the other issues